utils/mock-data-loader/portal-users/index.js

```diff
@@ -1,239 +1,279 @@
+  password: 'AbC!2345',
+  firstname: 'Reid',
+  surname: 'Oakley',
+  email: 'readonly2@ukexportfinance.gov.uk',
+  timezone: 'Europe/London',
+  roles: [ROLES.READ_ONLY],
+  bank: UKEF_TEST_BANK_2,
+  isTrusted: false,
+};
+
+export const ADMIN = {
+  username: 'test2@ukexportfinance.gov.uk',
+  password: 'AbC!2345',
+  firstname: 'Julius',
+  surname: 'No',
+  email: 'test2@ukexportfinance.gov.uk',
+  timezone: 'Europe/London',
+  roles: [ROLES.ADMIN, ROLES.MAKER],
+  bank: {
+    id: '*',
   },
-  READ_ONLY: {
-    username: 'readonly@ukexportfinance.gov.uk',
-    password: 'AbC!2345',
-    firstname: 'Reid',
-    surname: 'Oakley',
-    email: 'readonly@ukexportfinance.gov.uk',
-    timezone: 'Europe/London',
-    roles: [ROLES.READ_ONLY],
-    bank: {
-      id: '*',
-    },
-    isTrusted: false,
+  isTrusted: false,
+};
+
+export const READ_ONLY = {
+  username: 'readonly@ukexportfinance.gov.uk',
+  password: 'AbC!2345',
+  firstname: 'Reid',
+  surname: 'Oakley',
+  email: 'readonly@ukexportfinance.gov.uk',
+  timezone: 'Europe/London',
+  roles: [ROLES.READ_ONLY],
+  bank: {
+    id: '*',
   },
-  ADMINNOMAKER: {
-    username: 'adminnomaker@ukexportfinance.gov.uk',
-    password: 'AbC!2345',
-    firstname: 'Julius',
-    surname: 'No',
-    email: 'adminnomaker@ukexportfinance.gov.uk',
-    timezone: 'Europe/London',
-    roles: [ROLES.ADMIN],
-    bank: {
-      id: '*',
-    },
-    isTrusted: false,
+  isTrusted: false,
+};
+
+export const ADMINNOMAKER = {
+  username: 'adminnomaker@ukexportfinance.gov.uk',
+  password: 'AbC!2345',
+  firstname: 'Julius',
+  surname: 'No',
+  email: 'adminnomaker@ukexportfinance.gov.uk',
+  timezone: 'Europe/London',
+  roles: [ROLES.ADMIN],
+  bank: {
+    id: '*',
   },
-  TEST_EMAIL_NO_GOV_NOTIFY: {
-    username: 'test_no_notify@ukexportfinance.gov.uk',
-    password: 'AbC!2345',
-    firstname: 'Wolfgang',
-    surname: 'Weisen',
-    email: 'test_no_notify@ukexportfinance.gov.uk',
-    timezone: 'Europe/London',
-    roles: [ROLES.MAKER],
-    bank: {
-      id: '*',
-    },
-    isTrusted: false,
+  isTrusted: false,
+};
+
+export const TEST_EMAIL_NO_GOV_NOTIFY = {
+  username: 'test_no_notify@ukexportfinance.gov.uk',
+  password: 'AbC!2345',
+  firstname: 'Wolfgang',
+  surname: 'Weisen',
+  email: 'test_no_notify@ukexportfinance.gov.uk',
+  timezone: 'Europe/London',
+  roles: [ROLES.MAKER],
+  bank: {
+    id: '*',
   },
+  isTrusted: false,
+};
+
+const MOCK_USERS = {
+  BANK1_MAKER1,
+  BANK1_MAKER2,
+  BANK1_MAKER3,
+  BANK1_READ_ONLY1,
+  BANK3_GEF_MAKER1,
+  BANK3_GEF_CHECKER1,
+  BANK1_CHECKER1,
+  BANK1_MAKER_CHECKER1,
+  BANK1_MAKER_CHECKER2,
+  BANK1_PAYMENT_REPORT_OFFICER1,
+  BANK1_MAKER_PAYMENT_REPORT_OFFICER1,
+  BANK2_PAYMENT_REPORT_OFFICER1,
+  BANK2_MAKER2,
+  BANK2_MAKER1,
+  BANK2_CHECKER1,
+  BANK2_READ_ONLY1,
+  ADMIN,
+  READ_ONLY,
+  ADMINNOMAKER,
+  TEST_EMAIL_NO_GOV_NOTIFY,
 };
 
-module.exports = MOCK_USERS;
+export default MOCK_USERS;
```

Comment:
Why are we exporting this twice? Currently the exports will look like
```
{
  default: {
    BANK1_MAKER1,
    BANK1_MAKER2,
    BANK1_MAKER3,
    BANK1_READ_ONLY1,
    BANK3_GEF_MAKER1,
    BANK3_GEF_CHECKER1,
    BANK1_CHECKER1,
    BANK1_MAKER_CHECKER1,
    BANK1_MAKER_CHECKER2,
    BANK1_PAYMENT_REPORT_OFFICER1,
    BANK1_MAKER_PAYMENT_REPORT_OFFICER1,
    BANK2_PAYMENT_REPORT_OFFICER1,
    BANK2_MAKER2,
    BANK2_MAKER1,
    BANK2_CHECKER1,
    BANK2_READ_ONLY1,
    ADMIN,
    READ_ONLY,
    ADMINNOMAKER,
    TEST_EMAIL_NO_GOV_NOTIFY,
  },
  BANK1_MAKER1,
  BANK1_MAKER2,
  BANK1_MAKER3,
  BANK1_READ_ONLY1,
  BANK3_GEF_MAKER1,
  BANK3_GEF_CHECKER1,
  BANK1_CHECKER1,
  BANK1_MAKER_CHECKER1,
  BANK1_MAKER_CHECKER2,
  BANK1_PAYMENT_REPORT_OFFICER1,
  BANK1_MAKER_PAYMENT_REPORT_OFFICER1,
  BANK2_PAYMENT_REPORT_OFFICER1,
  BANK2_MAKER2,
  BANK2_MAKER1,
  BANK2_CHECKER1,
  BANK2_READ_ONLY1,
  ADMIN,
  READ_ONLY,
  ADMINNOMAKER,
  TEST_EMAIL_NO_GOV_NOTIFY,
}
```

Reply:
There are inconsistent imports for this, for example, some use this 

```js
import { BANK1_MAKER1 } from '..';
```

Others use this:

```js
const MOCK_USERS = require('..');
```

Some files have been updated to use `import` instead of `require` and having this file using `module.exports` was causing issues

Reply:
I still think this is a bit weird

The latter would be importing what I listed above, including a weird `MOCK_USERS.default` property. I think not using the export default & fixing the imports makes the most sense?

Reply:
i.e. you can get rid of `export default MOCK_USERS;` here. AFAIK there are 5 cases for importing the users

- `import { BANK1_MAKER1 } from '..';`  - this will import as intended
- `const { BANK1_MAKER1 } = require('..')` - will continue to work as intended
- `import MOCK_USERS from '..'` - **this will not work anymore** but it wont have worked before. If anywhere is doing this it should be updated 
- `import * as MOCK_USERS from '..'` - will continue to work
- ``const MOCK_USERS = require('..')` - will continue to work (I think! Slightly less confident on this)